tests/end2end/playwright/print.spec.js

```diff
@@ -710,4 +710,14 @@ test.describe('Error while printing', () => {
 
         await expect(page.locator("#message > div:last-child")).toHaveClass(/alert-danger/);
     });
+
+    test('Remove print overlay when switching to another minidock', async ({ page }) => {
+        await page.locator('#button-print').click();
+
+        await page.locator('#button-selectiontool').click();
+
+        await page.waitForTimeout(200); // wait to be sure tooltip is fully shown
+
+        expect(await page.locator('#newOlMap').screenshot()).toMatchSnapshot('print-selection.png');
```

Comment:
From our past experiences, doing screenshot comparaison is very not reliable. It has a high probability to be broken and to not work (to have the exact same pixel match across all computers and CI).

And looking at your screenshot, it looks a quite complex screenshot (OSM, text, CSS etc...).

This screenshot contains OSM data in the background, so if the OSM style change, or if the data is edited, this screenshot will be outdated and will break the test ?

Is-it not possible to avoid a screenshot comparaison ? By inspecting the OL map `toollayer` ?

Reply:
As we can notice, the CI is failing because of the screenshot comparaison ;-)

Reply:
Replaced it by counting how many canvas there is.